annotation-utils/pom.xml

```diff
@@ -1,333 +1,413 @@
-			<artifactId>truevfs-driver-zip</artifactId>
-			<version>0.12.0</version>
-		</dependency>
-
-		<!-- for managing git repositories -->
-		<dependency>
-			<groupId>org.eclipse.jgit</groupId>
-			<artifactId>org.eclipse.jgit</artifactId>
-		</dependency>
-
-		<!-- for extracting statistics from text -->
-		<dependency>
-			<groupId>org.nield</groupId>
-			<artifactId>kotlin-statistics</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.jetbrains.kotlinx</groupId>
-			<artifactId>kotlinx-html-jvm</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.jsoup</groupId>
-			<artifactId>jsoup</artifactId>
-		</dependency>
-
-	</dependencies>
-
-	<profiles>
-		<profile>
-			<id>run-jcasgen</id>
-			<activation>
-				<file>
-					<exists>src/main/resources/META-INF/org.apache.uima.fit/types.txt</exists>
-				</file>
-			</activation>
-			<build>
-				<plugins>
-					<plugin>
-						<!--generate types dynamically -->
-						<groupId>org.apache.uima</groupId>
-						<artifactId>jcasgen-maven-plugin</artifactId>
-						<version>2.9.0</version>
-						<configuration>
-							<limitToProject>true</limitToProject>
-							<typeSystemIncludes>
-								<include>src/main/resources/edu/isi/vista/annotationutils/uimafit/typeSystemDescriptor.xml</include>
-							</typeSystemIncludes>
-						</configuration>
-						<executions>
-							<execution>
-								<!--call it in the generate-source phase -->
-								<phase>generate-sources</phase>
-								<goals>
-									<goal>generate</goal>
-								</goals>
-							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<groupId>org.codehaus.mojo</groupId>
-						<artifactId>build-helper-maven-plugin</artifactId>
-						<version>1.10</version>
-						<executions>
-							<execution>
-								<id>addToSourceFolder</id>
-								<goals>
-									<!--add the generated sources -->
-									<goal>add-source</goal>
-								</goals>
-								<phase>process-sources</phase>
-								<configuration>
-									<sources>
-										<!--default path to generated sources -->
-										<source>${project.build.directory}/generated-sources/jcasgen</source>
-									</sources>
-								</configuration>
-							</execution>
-						</executions>
-					</plugin>
-					<plugin>
-						<groupId>org.codehaus.mojo</groupId>
-						<artifactId>appassembler-maven-plugin</artifactId>
-						<configuration>
-							<programs>
-								<program>
-									<mainClass>edu.isi.vista.annotationutils.PushAnnotationsKt</mainClass>
-									<id>pushAnnotations</id>
-								</program>
-							</programs>
-						</configuration>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
-
-	<!-- for "exposed", the Kotlin SQL library.  Not sure why it isn't on Maven Central... -->
-	<repositories>
-		<repository>
-			<id>jcenter</id>
-			<name>jcenter</name>
-			<url>http://jcenter.bintray.com</url>
-		</repository>
-	</repositories>
-
-</project>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Were there any substantive changes here or is this just reformatting?

Reply:
There was a change to a url (that Liz helped me fix) that prevented maven from creating the build. However, the large diff should be due to reformatting.